scripts/tests

```diff
@@ -388,17 +384,31 @@ def kat(force_qemu, verbose, cflags, arch_flags):
     type=bool,
     help="Benchmarking binary is run with sudo.",
 )
+@click.option(
+    "-w",
+    "--exec-wrapper",
```

Comment:
This seems to work only for individual commands, not for e.g. `--exec-wrapper "bash my_script.sh"` or `--exec-wrapper "my_wrapper arg0 arg1"`. It would be nice if it could work roughly like `xargs`.